diff --git a/node_modules/flutterwave-react-native/dist/FlutterwaveCheckout.js b/node_modules/flutterwave-react-native/dist/FlutterwaveCheckout.js
index 952c75c..0fa676c 100644
--- a/node_modules/flutterwave-react-native/dist/FlutterwaveCheckout.js
+++ b/node_modules/flutterwave-react-native/dist/FlutterwaveCheckout.js
@@ -105,7 +105,7 @@ var FlutterwaveCheckout = function FlutterwaveCheckout(props) {
         inputRange: [0, 0.3, 1],
         outputRange: [0, 1, 1]
     });
-    return (<Modal transparent={true} animated={false} hardwareAccelerated={false} visible={show}>
+    return (<Modal transparent={true} animationType="fade" hardwareAccelerated={false} visible={show}>
       <FlutterwaveCheckoutBackdrop onPress={function () { return handleAbort(); }} animation={animation.current}/>
       <Animated.View style={[
         styles.webviewContainer,
